pkg/api/controller_test.go

```diff
@@ -2311,8 +2311,8 @@ func TestController_DeleteBranchHandler(t *testing.T) {
 		if err != nil {
 			t.Fatal("DeleteBranch error:", err)
 		}
-		if resp.JSONDefault == nil {
-			t.Fatal("DeleteBranch expected error while trying to delete default branch")
+		if resp.StatusCode() != http.StatusForbidden {
```

Comment:
Test `resp.JSON403 == nil` will check the status code and the content type as the generated code will populate this field just for this case.